sections_flask-jwt-extended/models/item.py

```diff
@@ -16,12 +17,21 @@ def __init__(self, name, price, store_id):
         self.store_id = store_id
 
     def json(self):
-        return {'name': self.name, 'price': self.price}
+        return {
+            'id': self.id,
+            'name': self.name,
+            'price': self.price,
+            'store_id': self.store_id
+        }
```

Comment:
I choose to return more info in the `json()` method so it's more convenient when testing.